ApplicationServerDeployment/libs/config.py

```diff
@@ -66,9 +74,19 @@ def _read_config(self):
 
             self.region_name = 'us-east-1' # Leave this as us-east-1 unless you know what you are doing
             self.web_app_url = ''
+            self.alexa_skill_id = config.get('Config', {}).get('ALEXA_SKILL_ID', 'ask.skill.1234567890')
```

Comment:
more to more appropriate method, this is specifically for Grafana

Reply:
This was defined under the non Grafana config method _read_config. Grafana configs are in method _read_grafana_config